.gitignore

```diff
@@ -1,3 +1,6 @@
+.rush
```

Comment:
Is this necessary if we use `yarn` for Spectral?

Reply:
It is not necessary. It's an artifact of how I was working on this changeset and I wasn't sure if it was worth including (potentially ignoring other IDE/tool directories such as `.idea` or similar). Happy to remove that if we want to keep the `.gitignore` cleaner 👍